src/shared/bg_mod.h

```diff
@@ -0,0 +1,93 @@
+/*
+===========================================================================
+
+Unvanquished GPL Source Code
+Copyright (C) 1999-2005 Id Software, Inc.
+Copyright (C) 2000-2009 Darklegion Development
+
+This file is part of the Unvanquished GPL Source Code (Unvanquished Source Code).
+
+Unvanquished is free software; you can redistribute it
+and/or modify it under the terms of the GNU General Public License as
+published by the Free Software Foundation; either version 2 of the License,
+or (at your option) any later version.
+
+Unvanquished is distributed in the hope that it will be
+useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Unvanquished; if not, write to the Free Software
+Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+===========================================================================
+*/
+
+struct meansOfDeathData_t
+{
+	meansOfDeath_t number;
+	const char     *name;
+};
+
+static const meansOfDeathData_t bg_meansOfDeathData[] =
+{
+	{ MOD_UNKNOWN, "MOD_UNKNOWN" },
```

Comment:
The table data should go in a .cpp file. Doing it like this in a header means that every file we build gets its own copy of the table and can cause [ODR](https://en.cppreference.com/w/cpp/language/definition) violations for code that uses it.